### PR TITLE
Tweak ProGuard rules

### PR DESCRIPTION
**Pull request by Goooler**
## Type of change

:white_check_mark: General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
:x: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: proguard.pro [hunk 1]
```diff
@@ -42,13 +42,14 @@
 -keep class app.lawnchair.LawnchairApp { *; }
 -keep class app.lawnchair.LawnchairLauncher { *; }
 -keep class app.lawnchair.compatlib.** { *; }
+
 -keep class com.google.protobuf.Timestamp { *; }
+
 # TODO: Remove this after the change in https://github.com/ChickenHook/RestrictionBypass/pull/9 has been released.
 -keep class org.chickenhook.restrictionbypass.** { *; }
--keep class androidx.compose.runtime.** { *; }
```
Comment (Goooler) [inline proguard.pro:48]: Is there any specific reason to keep all classes under this path?

Comment (MrSluffy) [reply proguard.pro:48]: Yes, or you can verify building in release?

Comment (Goooler) [reply proguard.pro:48]: I don't see any crash in Settings screen after removing this line, where you saw?

Comment (MrSluffy) [reply proguard.pro:48]: No, I mean you successfully build in release build right? And no crash?. I forgot why I added, but if you don't faced any issue in release build that's great. 

Anyway will check it

Comment (Goooler) [reply proguard.pro:48]: I checked, nothing crashed, just keep the removing now.

**Review by MrSluffy — APPROVED** [review]

**Pull request merged**